plugin-build/src/main/kotlin/io/sentry/android/gradle/util/SentryAndroidSdkChecker.kt

```diff
@@ -3,6 +3,7 @@ package io.sentry.android.gradle.util
 import io.sentry.android.gradle.services.SentryModulesService
 import org.gradle.api.Project
```

Comment:
`l`: Maybe rename the class to something like `SentryModulesCollector`?

Reply:
The function name we could rename from `detectSentryAndroidSdk` to something like `collectModules` or so?